.github/workflows/base-lambdas-reusable-deploy-all.yml

```
@@ -640,3 +640,17 @@ jobs:
       lambda_layer_names: "core_lambda_layer"
     secrets:
       AWS_ASSUME_ROLE: ${{ secrets.AWS_ASSUME_ROLE }}
+
+  deploy_dynamodb_migration_lambda:
+    name: Deploy DynamoDB Migration Lambda
+    uses: ./.github/workflows/base-lambdas-reusable-deploy.yml
+    with:
+      environment: ${{ inputs.environment }}
+      python_version: ${{ inputs.python_version }}
+      build_branch: ${{ inputs.build_branch }}
+      sandbox: ${{ inputs.sandbox }}
+      lambda_handler_name: migration_dynamodb_handler
+      lambda_aws_name: MigrationDynamoDBLambda
```

As commented on the infra PR, we should avoid mention of resource within name
```suggestion
      lambda_aws_name: MigrationDynamoDB
```